src/styles/Create-Feedback.scss

```diff
@@ -0,0 +1,204 @@
+@import 'mixins.scss';
+
+.create-feedback-page {
+  background-color: #f2f2f2;
```

Comment:
Suggestion to set variables for these CSS colors. Not a requirement, but it is good practice to do so.

Reply:
I had them as variables, but didn't use them. Updated!